akka-persistence-typed/src/main/scala/akka/persistence/typed/LwwTime.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed
+
+/**
+ * Utility class for comparing timestamp and data center
+ * identifier when implementing last-writer wins.
+ */
+final case class LwwTime(timestamp: Long, originDc: ReplicaId) {
```

Comment:
This is a duplucation of `akka.persistence.typed.crdt.LwwTime`. I will remove it.